balancer-js/src/types.ts

```diff
@@ -282,6 +282,7 @@ export enum PoolType {
 export interface Pool {
   id: string;
   name: string;
+  athRateProduct?: string;
```

Comment:
Guessing this was from Luizs code and might want to remove to keep consistency.

Reply:
True - missed that one 😅